src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsSelector.js

```diff
@@ -259,10 +260,15 @@ function MoneyRequestParticipantsSelector({
     );
 
     // When search term updates we will fetch any reports
```

Comment:
Can we also update this comment to explain the new changes?

Reply:
Also since this is pretty much copy / pasted in 3 files, should we make it a reusable lib?

Reply:
Yes, I think we should do both. 

cc: @Krishna2323

Reply:
We extract this
```
    const setSearchTermAndSearchInServer = useCallback(
        (text = '') => {
            if (text && !maxParticipantsReached) {
                Report.searchInServer(text);
            }
            setSearchTerm(text);
        },
        [maxParticipantsReached],
    );
```
 to a function in `OptionListUtils.ts` (or a more appropriate file) and use it in the three pages. @Krishna2323

Reply:
How about a custom hook like this?

```js
import { useCallback } from 'react';
import * as Report from '@userActions/Report';

const useSearchTermAndSearch = (setSearchTerm, maxParticipantsReached) => {
    const setSearchTermAndSearchInServer = useCallback(
        (text = '') => {
            if (text && !maxParticipantsReached) {
                Report.searchInServer(text);
            }
            setSearchTerm(text);
        },
        [maxParticipantsReached, setSearchTerm]
    );

    return setSearchTermAndSearchInServer;
};

export default useSearchTermAndSearch;
```
and use it in the pages like this
```js
const setSearchTermAndSearchInServer = useSearchTermAndSearch(setSearchTerm, maxParticipantsReached);
```
WDYT? @Beamanator